src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -421,6 +429,65 @@ public function getXEditableChoices(FieldDescriptionInterface $fieldDescription)
         return $xEditableChoices;
     }
 
+    /**
+     * Returns a canonicalized locale for "moment" NPM library,
+     * or `null` if the locale's language is "en", which doesn't require localization.
+     *
+     * @return null|string
+     */
+    final public function getCanonicalizedLocaleForMoment(array $context)
+    {
+        $locale = strtolower(str_replace('_', '-', $context['app']->getRequest()->getLocale()));
+
+        // "en" language doesn't require localization.
+        if ('en' === $lang = substr($locale, 0, 2)) {
+            return null;
+        }
+
+        // `moment: ^2.8` only ships "es" and "es-do" locales for "es" language
+        if ('es' === $lang && !in_array($locale, ['es', 'es-do'], true)) {
```

Comment:
This poses the same problem for `nl-nl`. Moment only has `nl.js`, but Symfony's request locale is `nl-nl`.

Reply:
Can you do something about it @phansys ?

Reply:
@svenluijten, @greg0ire: https://github.com/sonata-project/SonataAdminBundle/pull/4878/commits/83484cdeed6e3d63a121a31f234e3a684875bcc8#diff-83a367a366e6d48e0d8ea08cea7f0351R451